example/pubspec.yaml

```diff
@@ -9,8 +9,6 @@ dependencies:
   flutter:
     sdk: flutter
 
-  cupertino_icons: ^0.1.2
```

Comment:
`cupertino_icons` was never used and the old version prevented running in null-safe mode.